### ISSUE TITLE
Update REP Token Contract Address -- First Week of July

### ISSUE THREAD

**Issue by tayvano** [ethereum-lists/tokens#84]
Hello,

We are writing to inform you that Augur is set to be deployed on the Ethereum main network the morning of July 9th, 2018. With deployment, we also must conduct a migration of the REP token
balances, so that the new REP token exists inside the Augur genesis universe.

On July 9th, 2018 at 11:01AM PST, the current REP token contract will be frozen indefinitely. Once
this occurs, no more transfers of the tokens on the old contract will be possible. We have publicly disclosed this on Wednesday, May 16th, 2018 in our ‘[Deployment Details & REP Migration](https://medium.com/@AugurProject/deployment-details-rep-migration-e5413ff9fb65)’ blog post.

In the hours following, we will be minting all REP token holders an equivalent amount of REP on the new REP contract, which will be the REP contract going forward.

There is no threat of loss of funds.
 
What we need from you:  

1. Change the REP contract address from the old one to the new one after we have confirmed a successful migration.

------------------------------------------------------------------

Contract Freeze Time: July 9th, 2017 @ 11:01AM PST

This is expected to be a smooth migration, and the Augur team is on-call to assist with anything you may need. We will follow up with you (5) five days prior to deployment day, the day of, and then after the migration is complete with the updated REP contract address.

Please confirm with us that you’ve received this message and that you intend to follow the steps above upon contract freeze time. We will follow up this email in (7) seven days if we do not hear back.

If this is not the correct point of contact, we would kindly appreciate the message routed to the appropriate personnel on your team.

Thanks,

Tom Kysar

**Comment by ligi** [comment]
hey @tayvano - be aware I will be offline the first week of juli (festival) - but you should be able to merge - but perhaps give it a try before so there is no problem in the pipeline then ..-)

**Comment by tayvano** [comment]
@ligi Thanks for letting me know. Definitely doable on my end. I created this issue just as much for me as for you. 😉

cc @409H so you can follow this too

**Comment by ligi** [comment]
I just returned - is this still a thing?

**Comment by tayvano** [comment]
Hey hey! Hope your vacay was great and relaxing!

Just waiting on the address from Augur right now

Latest info:

```
a) New augur.net website launched today - check it out for an overview of Augur, the Forecast Foundation, bug bounties, and frequently asked questions. 

b) REP migration is planned to begin at 11:01AM PT tomorrow, July 9th. At that time, the current REP contract will be frozen indefinitely. The migration will take place over the rest of the day. We do not have an exact estimation of how long the migration will take, this channel & Twitter will be updated during the process tomorrow.

c) Prior Augur App releases have been removed, the final release will be published tomorrow once the REP migration is complete. Blog posts coming in the morning. 
```

**Comment by tayvano** [comment]
> Hello, 

> We are writing to let you know that the REP token migration is confirmed to begin at 11:01AM PT. At this time, the current REP contract will be frozen indefinitely. 

> Current REP contract that will be frozen: 0xe94327d07fc17907b4db788e5adf2ed424addff6

> https://medium.com/@AugurProject/rep-migration-deployment-122e7189779d

> We will be conducting the automatic migration immediately after. We expect this process to take around 8 to 10 hours (higher end of the spectrum estimate) assuming all goes smoothly. The previous REP migration took about 3.5 hours, and there are now about double the amount of unique accounts that hold REP this time around. 

> I will follow up to let you know the new REP token contract address as soon as we have confirmed that the migration is complete and successful. Feel free to reach out to us at the @AugurProject twitter (DM), or join our Discord and DM (@tk) for any real-time communications or questions, or just reply to this email thread. 

> Thanks, 

> Tom

Forecast Foundation OU
tom@augur.net

**Comment by tayvano** [comment]
> Hello, 

> The migration is complete, and Augur is deployed. 

> New REP token contract address: 0x1985365e9f78359a9B6AD760e32412f4a445E862

> Announcement: https://medium.com/@AugurProject/augur-launches-794fa7f88c6a

> Thank you for the cooperation during the migration. You can now update to the new REP token contract address above. 

> Please let me know if you have any questions.

> Cheers,

> The Forecast Foundation OU

Will open a PR to update in the AM. In bed atm.

**Comment by ligi** [comment]
cool - thanks for all the detailed info - will make the PR now - just not sure what to do with the old address - perhaps do a REPOLD token?
This seems to be something that happens often - perhaps we should even make a new field in the model - e.g. a boolean deprecated. So that we can better show in the UI that a token is deprecated.

**Comment by ligi** [comment]
cc @409H @gamalielhere @vikmeup - what do you think about a deprecated field in the token lists? Perhaps this should even have a field that links it to the new token?

**Comment by tayvano** [comment]
Agreed. 

Digix (https://github.com/ethereum-lists/tokens/pull/102) now has two tokens as well due to contract upgrade. It should be noted that updating or changing an existing token like in these cases is not the same as duplicate symbols by different projects (which is another issue entirely)

I think there are actually three REP tokens now:

- First one that was used 2015/2016/2017? Locked a few months back due to Serpent bug

- Second one that was used until today

- This third one that was just deployed
